src/natcap/invest/wind_energy.py

```diff
@@ -949,144 +972,158 @@ def execute(args):
         task_name='mask_depth_on_bathymetry',
         dependent_task_list=depth_mask_dependent_task_list)
 
-    # Set paths for creating density and harvested rasters
-    temp_density_raster_path = os.path.join(
-        inter_dir, 'temp_density%s.tif' % suffix)
-    temp_harvested_raster_path = os.path.join(
-        inter_dir, 'temp_harvested%s.tif' % suffix)
+    align_dependent_task_list = [create_depth_mask_task]
+    rasters_to_align_list = [depth_mask_path]
+    aligned_depth_mask_path = depth_mask_path.replace(
+        '%s.tif' % suffix, '_aligned%s.tif' % suffix)
+    aligned_rasters_list = [aligned_depth_mask_path]
+
+    # If we will be masking by distance, align the distance raster as well
+    if mask_by_distance:
+        # Mask the distance raster by the min and max distances
+        LOGGER.info('Creating Distance Mask')
+        dist_mask_path = os.path.join(inter_dir,
+                                      'distance_mask%s.tif' % suffix)
+        create_dist_mask_task = task_graph.add_task(
+            func=_mask_by_distance,
+            args=(dist_trans_path, min_distance, max_distance,
+                  _TARGET_NODATA, dist_mask_path),
+            target_path_list=[dist_mask_path],
+            task_name='mask_raster_by_distance',
+            dependent_task_list=[create_distance_raster_task])
+
+        align_dependent_task_list.append(create_dist_mask_task)
+        rasters_to_align_list.append(dist_mask_path)
+        aligned_dist_mask_path = dist_mask_path.replace(
+            '%s.tif' % suffix, '_aligned%s.tif' % suffix)
+        aligned_rasters_list.append(aligned_dist_mask_path)
+    else:
+        # No mask_by_distance_task is added to taskgraph
+        LOGGER.info('NO Distance Mask to add to list')
+        # No need to align bathy
+
+    if not run_valuation:
+        if mask_by_distance:
```

Comment:
We don't want to do this regardless of `run_valuation` because we'll have to do it again if running valuation. I'll check that assumption below.